CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
   - `processorhelper.New[Traces|Metrics|Logs]ProcessorWithCreateSettings`
   - `component.NewExtensionFactoryWithStabilityLevel`
 
+### 💡 Enhancements 💡
+
+- Add `skip-get-modules` builder flag to support isolated environment executions (#6009)
```

Comment:
I think you need an extra new line after your new entry and before the previous release version.

Reply:
Thanks, fixed it.